step.yml

```diff
@@ -0,0 +1,38 @@
+---
+title: Generate Xcode html report
```

Comment:
I think this title isn't really explaining what the step does, what about something like `Visualise Xcode test results` or `Generate Xcode test report html?` ?

Reply:
`Generate Xcode test report html` sounded better to me so I chose that.